releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml

```diff
@@ -35,6 +35,7 @@ upgrade:
   - |
```

Comment:
?

Reply:
Odd, I must have opened and my editor did magic. Fixed in 77babd399

Reply:
I still see three unrelated notes with diffs

Reply:
I checked out from master on all of these. For the first 2 no files were updated (even though Github is marking them here). The final one did have a minor update.

Reply:
Fixed in f9ec51310